tests/integrationv2/providers.py

```diff
@@ -139,6 +139,10 @@ def setup_server(self, options: ProviderOptions):
             cmd_line.append('--insecure')
         if options.tls13 is True:
             cmd_line.append('--tls13')
+        if options.use_client_auth is True:
```

Comment:
`is True` probably isn't necessary, but does looks like the convention in the file